content/_data/changelogs/weekly.yml

```diff
@@ -14157,6 +14157,169 @@
         authors:
           - Wadeck
 
+  - version: '2.335'
+    date: 2022-02-13
+    changes:
+      - type: TODO
+        category: developer
+        pull: 6262
+        authors:
+          - dependabot[bot]
+          - basil
+        pr_title: 'Winstone 5.22: Bump Jetty from 9.4.43.v20210629 to 9.4.45.v20220203'
+        message: |-
+          Winstone 5.22: Bump Jetty from 9.4.43.v20210629 to 9.4.45.v20220203
+             Winstone 5.22 changelog: https://github.com/jenkinsci/winstone/releases/tag/winstone-5.22
+               Append to log file rather than truncating it (https://github.com/jenkinsci/winstone/pull/200)
+               Write log file and access logs in UTF-8 encoding rather than default encoding (https://github.com/jenkinsci/winstone/pull/200)
+             Jetty 9.4.45 changelog: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.45.v20220203
+      - &1
+        type: major rfe
+        category: major rfe
+        pull: 5923
+        authors:
+          - janfaracik
+        pr_title: Modernise form components
+        message: |-
+          Modernise form components.
+      - *1
```

Comment:
Needs to be removed.

Reply:
@MarkEWaite action item after release is available